docs/admin/runai-setup/cluster-setup/cluster-install.md

```diff
@@ -1,11 +1,11 @@
-Below are instructions on how to install a Run:ai cluster. 
+Below are instructions on how to install a Run:ai cluster.
 
 ## Prerequisites
-Before installing, please review the installation prerequisites here: [Run:ai GPU Cluster Prerequisites](cluster-prerequisites.md). 
+Before installing, please review the installation prerequisites here: [Run:ai GPU Cluster Prerequisites](cluster-prerequisites.md).
 
 
 !!! Important
-    We strongly recommend running the Run:ai [pre-install script](cluster-prerequisites.md#pre-install-script) to verify that all prerequisites are met. 
+We strongly recommend running the Run:ai [pre-install script](cluster-prerequisites.md#pre-install-script) to verify that all prerequisites are met.
```

Comment:
@roir This will not render as an admonition. The indentation is needed to tell markdown what to do.
```suggestion
    We strongly recommend running the Run:ai [pre-install script](cluster-prerequisites.md#pre-install-script) to verify that all prerequisites are met.
```